packages/terra-application-utility/src/utility/_UtilityButton.jsx

```diff
@@ -41,6 +41,10 @@ const propTypes = {
    * The image associated with utilities.
    */
   accessory: PropTypes.element,
+  /**
+   * Role attribute of Utlity Menu.
```

Comment:
Can we change this to 
```suggestion
   * The role attribute to set on the menu.
```

Reply:
sure updated in all the above mentioned [places](https://github.com/cerner/terra-framework/pull/501/commits/95bc33f8a3c45d78146c62cb9d9add2de6f13b71).